pkg/operator/ceph/cluster/controller.go

```diff
@@ -529,6 +564,27 @@ func (c *ClusterController) onUpdate(oldObj, newObj interface{}) {
 		}
 		return
 	}
+
+	// Display success after upgrade
+	if versionChanged {
+		versions, err := client.GetCephVersions(c.context, cluster.Namespace)
+		if err != nil {
+			logger.Errorf("failed to get ceph daemons versions. %+v", err)
+			return
+		}
+
+		if len(versions.Overall) == 1 {
+			for v := range versions.Overall {
```

Comment:
We have already checked here that there is only one element in versions.Overall, so range just adds extra indentation without purpose.

Reply:
The problem is that we don't know the name of the key so we can access it, here the key is value... So unless we do this iteration I don't know how to access it.